packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx

```diff
@@ -98,10 +98,12 @@ export const PluginVerifiedWalletStatusBar = memo<PluginVerifiedWalletStatusBarP
                 providerIcon: account ? providerDescriptor?.icon : undefined,
                 iconFilterColor: account ? providerDescriptor?.iconFilterColor : '',
                 formattedAddress: Others?.formatAddress(account || (defaultVerifiedWallet?.identity ?? ''), 4),
-                addressLink: Others?.explorerResolver.addressLink?.(
-                    account ? chainId : defaultChainId,
-                    account || (defaultVerifiedWallet?.identity ?? ''),
-                ),
+                addressLink:
```

Comment:
Please revert this change. `addressLink` should be a URL and should not use account.

Reply:
you tested?

Reply:
```
 <Link
                            href={addressLink}
                            target="_blank"
                            title="View on Explorer"
                            rel="noopener noreferrer"
                            onClick={(event) => {
                                event.stopPropagation()
                            }}
                            className={classes.linkIcon}>
                            <Icons.LinkOut size={14} className={classes.linkIcon} />
                        </Link>
```

So you want to set an address to `href`? Do you think that makes sense?

Reply:
And test result: 
![image](https://user-images.githubusercontent.com/28348152/182994456-37602128-3957-45a5-9b41-d3e4d1d13bdd.png)

![image](https://user-images.githubusercontent.com/28348152/182994496-735e29bc-5aed-4b96-91f8-a81e90756448.png)